justfile

```diff
@@ -62,6 +62,9 @@ run firmware=default:
 build config:
 	cargo run --package runner -- build --config {{config}}
 
+build-firmware config firmware=default:
```

Comment:
In a `justfile` it is important to add a comment on top of each top-level rule: it is display as documentation when running `just --list` (which is our default command). You can try to run `just` without any argument to see the current list of commands.

I realize that we are missing documentation on some other commands, so I'll add them too in the future :)